client/Contexts/MusicContext.tsx

```diff
@@ -110,6 +110,10 @@ export const MusicContextProvider: React.FC<MusicContextProps> = (props) => {
         _playSong(nextSong);
     }, [_playSong, songQueue, songQueueIndex, userQCount]);
 
+    const toggleFlag = useCallback(() => {
```

Comment:
Is there any specific reason this is a `useCallback`?

Reply:
not really, I removed it in the new commit